src/pyop/provider.py

```diff
@@ -57,9 +59,11 @@ def __init__(self, signing_key, configuration_information, authz_state, clients,
         :param clients: see <a href="https://openid.net/specs/openid-connect-registration-1_0.html#ClientMetadata">
             "OpenID Connect Dynamic Client Registration 1.0", Section 2</a>
         :param userinfo: read-only interface for user info
+        :param stateless: users, authorization codes, and tokens related data is not persisted
         :param id_token_lifetime: how long the signed ID Tokens should be valid (in seconds), defaults to 1 hour
         """
         self.signing_key = signing_key
+        self.encryption_key = encryption_key
```

Comment:
`encryption_key` doesn't seem to be used by the `Provider`

Reply:
removed